feature-libs/product/future-stock/styles/_future-stock-accordion.scss

```diff
@@ -0,0 +1,27 @@
+cx-future-stock-accordion {
+	display: block;
+  overflow: hidden;
+
+	.cx-future-stock-accordion-header {
+		border: none;
+		background-color: transparent;
+		padding-left: 20px;
```

Comment:
Our pipeline does not allow using properties like: 'padding-left', 'padding-right', '-top', '-bottom' (same for margins and borders). You need to specify the whole `padding` property, otherwise, pipeline validation will reject your PR.

Reply:
done